src/inmanta/execute/runtime.py

```diff
@@ -18,7 +18,7 @@
 from abc import abstractmethod
 from typing import TYPE_CHECKING, Deque, Dict, Generic, Hashable, List, Optional, Set, TypeVar, Union, cast
 
-import inmanta.ast.attribute  # noqa: F401 (pep8 does not recognize partially qualified access ast.attribute)
+import inmanta.ast.attribute  # noqa: F401 (pyflakes does not recognize partially qualified access ast.attribute)
```

Comment:
@wouterdb any opinion on this? I don't like the `noqa`, and I'm not sure about the practice in the first place, but I do really prefer `ast.attribute.RelationAttribute` over both `inmanta.ast.attribute.RelationAttribute` and `attribute.RelationAttribute`. Is this too exotic?